lib/poller.js

```diff
@@ -24,24 +28,66 @@ const LOGS_CLEAR_INTERVAL = 1 * T.day
 const SANCTIONS_INITIAL_DOWNLOAD_INTERVAL = 5 * T.minutes
 const SANCTIONS_UPDATE_INTERVAL = 1 * T.week
 const RADAR_UPDATE_INTERVAL = 5 * T.minutes
-
 const CHECK_NOTIFICATION_INTERVAL = 20 * T.seconds
-
 const PENDING_INTERVAL = 10 * T.seconds
+const CACHE_ENTRY_TTL = 3600 // seconds
```

Comment:
Should just use the available `T` functions here
```suggestion
const CACHE_ENTRY_TTL = 1 * T.hour
```

Reply:
This library uses seconds as the base unit, not milliseconds so using 1 * T.hour here would actually make the TTL 1000 * 3600 seconds long